langchain/src/storage/tests/convex/convex/test.ts

```diff
@@ -0,0 +1,45 @@
+import { v } from "convex/values";
```

Comment:
Rename to `lib.ts` or something similar, I think this will get picked up by the test runner